insights/specs_default.py

```diff
@@ -469,6 +469,9 @@ def block(broker):
     systemctl_list_unit_files = simple_command("/bin/systemctl list-unit-files")
     systemctl_list_units = simple_command("/bin/systemctl list-units")
     systemctl_mariadb = simple_command("/bin/systemctl show mariadb")
+    systemctl_pulp_workers = simple_command("/bin/systemctl show pulp_workers")
+    systemctl_pulp_resmg = simple_command("/bin/systemctl show pulp_resource_manager")
+    systemctl_pulp_celerybeat = simple_command("/bin/systemctl show pulp_celerybeat")
```

Comment:
Need to confirm if it's necessary to add these 'simple_command' to "specs_insights_archive.py” manually. Let's wait for a while to merge this PR.

Reply:
@TripleJQK - please add the corresponding `simple_file` according to the steps mentioned in the weekly meeting.